app/api/dao/user.py

```diff
@@ -43,6 +56,17 @@ def create_user(data):
     @staticmethod
     @email_verification_required
     def delete_user(user_id):
+        """ Deletes a user.
+        
+        Deletes the specified user and removes them from the directory, with checks to make sure that the user exists and is not the only administrator.
+        
+        Arguments:
+            user_id: The ID of the user to be deleted.
+            
+        Returns: 
+            A list with two elements. The first element is a dictionary containing a key 'message' containing a string which indicates whether or not the user was created successfully. The second is the HTTP response code.
```

Comment:
```suggestion
            A tuple with two elements. The first element is a dictionary containing a key 'message' containing a string which indicates whether or not the user was created successfully. The second is the HTTP response code.
```

What is returned is a tuple and not a list.